paper.md

```diff
@@ -0,0 +1,400 @@
+
+i.e., What does this software do?
+-->
+
+# Summary
+
+Empirical is a C++ library designed to promote open science and facilitate the development of scientific software that is efficient, reliable, and easily distributable to researchers and non-experts alike.
+Specifically, the library sets out to fulfill the following goals:
+
+1. **Utility:** Empirical features a wide selection of helper tools to streamline common scientific computing tasks such as configuration, data management, random number generation, and mathematical manipulations. Documentation, examples, and project templates streamline developer onboarding.
+2. **Efficiency:** Empirical tools emphasize efficiency to make heavy scientific computing workloads tractable. Where possible, computation is moved to compile-time, heap allocation is avoided, and memory-locality is respected.
+3. **Reliability:** Scientific simulation and data analysis is meaningful insofar as it is correct. In conjunction with extensive unit testing, Empirical benefits from --- and provides --- sophisticated debug-mode instrumentation tools. These include audited memory management, drop-in replacements for standard C++ containers with safety checking, and self-documenting assertions.
+4. **Distributability:** Scientific software should be easily used by any researcher, student, or citizen scientist who wants to experiment, explore, or replicate studies. As such, Empirical is highly portable, uses common data formats, facilitates flexible runtime configuration, and simplifies compilation to a performant web app.
+
+# Statement of Need
+<!-- STATEMENT OF NEED
+JOSS: illustrates the research purpose of the software.
+JOSS reviewer checklist: "clearly state what problems the software is designed to solve and who the target audience is"
+-->
+
+Modern web-based interfaces give computational research the unique potential to embody open science objectives of making the scientific process more transparent with auditable and exensible code, clear and replicable methodologies, and production of accessible results [CITE osf.io].
+<!-- [@world_academy_of_science_engineering_and_technology_2019] -->
+In practice, however, many scientific software applications are difficult to obtain, install, or use, and produce data in propriatary formats.
+
+High quality open-science tools encourage researchers to follow effective software development practices by simplifying development and helping them improve code quality, scientific rigor, and ease of replication or extension, resulting in broader confidence in and adoption of their software.
+In the process, researchers get more out of their own code.
+For example, an effective toolkit would allow a developer to easily add a GUI that would not only help other users of the software, but could also help the researcher themsleves gain "soft knowledge" of their system [@10.1145/3185517].
```

Comment:
I would cut "an effective toolkit would allow a developer to easily add a" and rewrite to just "For example, adding a GUI not only helps other uses of the software, but could also help..."